### ISSUE TITLE
Do not remove comments from values.yaml in UI

### ISSUE THREAD

**Issue by omercnet** [rancher/dashboard#2184]
When adding an app/helm the comments from the original values.yaml are removed

Many chart maintainers provide helpful guidance in the form of comments, removing them makes it challenging to deploy new charts.

**Comment by vincent99** [comment]
I agree but this would require backend changes as the values are currently delivered as already parsed into JSON and the comments are gone at that point.
